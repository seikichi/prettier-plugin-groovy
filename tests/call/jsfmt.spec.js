run_spec(__dirname, ["groovy"]);
