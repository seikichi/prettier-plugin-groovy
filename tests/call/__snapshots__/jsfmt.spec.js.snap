// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`helloworld.groovy 1`] = `
println "Hello, World!"
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
println "Hello, World!"

`;
